rasa/nlu/training_data/formats/luis.py

```diff
@@ -18,13 +18,13 @@ def read_from_json(self, js: Dict[Text, Any], **kwargs: Any) -> "TrainingData":
         training_examples = []
         regex_features = []
 
-        # Simple check to ensure we support this luis data schema version
-        if not js["luis_schema_version"].startswith("2"):
-            raise Exception(
-                "Invalid luis data schema version {}, "
-                "should be 2.x.x. "
-                "Make sure to use the latest luis version "
-                "(e.g. by downloading your data again)."
+        luisSchemaVersionChecked = 4
+        version = int(js["luis_schema_version"].split(".")[0])
+        if version > luisSchemaVersionChecked:
+            logger.warning(
```

Comment:
@imLew That's also a `warnings.warn`, correct?

Reply:
It is, in the sense that the user can theoretically fix it, but we would be telling them to downgrade their LUIS version. So I think this is more in the category of things the user should be aware of rather than things they should fix.

> @imLew You have to merge, I think @LucasPenido does not have the permissions to do so

I wanted to but it was blocked at the time by the broken temp file cleanup which caused the tests to fail.